test/common/bitmap_test.cpp

```diff
@@ -0,0 +1,113 @@
+#include <bitset>
+#include <thread>
+#include <unordered_set>
+
+#include "gtest/gtest.h"
+#include "common/test_util.h"
+#include "common/container/bitmap.h"
+
+namespace terrier {
+template<uint32_t num_elements>
+void CheckReferenceBitmap(const common::RawBitmap &tested,
```

Comment:
Maybe again put this function into test util, since it is now used in two places.